### PR TITLE
feat: #2943 Update the app aside in the marketplace developer preview

### PR DESCRIPTION
**Pull request by Bdragon93**
# Pull request checklist

fixes: #2943 

# Screenshot

Screenshot 2020-11-23 at 4 42 39 PM
Screenshot 2020-11-23 at 4 43 50 PM

### CODE REVIEW

File: packages/developer-portal/src/components/pages/app-detail-preview/client/contact-developer-modal.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react'
```
Comment (willmcvay) [inline packages/developer-portal/src/components/pages/app-detail-preview/client/contact-developer-modal.tsx:1]: We can delete this component now I think

File: packages/marketplace/src/components/pages/app-detail/client/contact-developer-modal.tsx [hunk 2]
```diff
@@ -24,91 +24,76 @@ export const closeContactModal = (setVisible: React.Dispatch<React.SetStateActio
...
-                  <a className={linkStyles.link} href={homePage} target="_blank" rel="noopener noreferrer">
-                    {homePage}
-                  </a>
-                )}
-              </p>
-            </GridItem>
-          </Grid>
-        </Content>
-      </ModalV2>
+      <Content>
```
Comment (willmcvay) [inline packages/marketplace/src/components/pages/app-detail/client/contact-developer-modal.tsx:31]: As per above, think can delete this now

**Review by willmcvay — COMMENTED** [review]
So part of the task is not just to render the sidebar links but also render the "About Developer" text in the main body of the page. Can't see any code to do this unless I have missed it :-)

File: packages/developer-portal/src/components/pages/app-detail-preview/client/contact-developer-modal.tsx (hunk 1 above)
Comment (Bdragon93) [reply packages/developer-portal/src/components/pages/app-detail-preview/client/contact-developer-modal.tsx:1]: I reused this component, just remove the modal the button to open it

**Review by willmcvay — APPROVED** [review]
LGTM 👍

**Pull request merged**